src/util.js

```diff
@@ -23,6 +23,38 @@ export function transparent(colors, opacity) {
   });
 }
 
+/**
+ * Returns a neighborhood around date that is between startDate and endDate.
+ * Default duration is the same as endDate - startDate.
+ *
+ * @param {Date} date
+ * @param {Date} startDate
+ * @param {Date} endDate
+ * @param {number|null} duration of neighborhood, in milliseconds
+ */
+export function computeNeighborhood(date, startDate, endDate, duration) {
+  if (duration == null) {
+    duration = endDate - startDate;
+  }
+  const neighborhood = { start: date - duration / 2, end: date.getTime() + duration / 2 };
```

Comment:
why do you call `getTime` in one but not in the other case? I think JavaScript will coalesce also the first case but the second case is more clear

Reply:
The getTime (or some cast to get milliseconds) is necessary because of the following '+' operator, which gets interpreted as string concatenation.   I tried duration / 2 + date, but that doesn't work either.  However, Number(date) + duration/2 does work.    It would be more consistent to use getTime() or Number() always, but that's annoying too.

Also, should this function return a Date instead of milliseconds?

Reply:
I get the reasoning behind `getTime`, what is confusing me is the inconsistency that when computing `start` you don't call `.getTime()` but when computing `end` you do.

Reply:
Ah, because the '-' operator is only numeric, so the date is auto converted to the number of milliseconds.